model/textparse/openmetricsparse.go

```diff
@@ -225,6 +225,10 @@ func (p *OpenMetricsParser) nextToken() token {
 	return tok
 }
 
+func parseError(exp string, got token) error {
```

Comment:
Can we have unified error formatting for OM, too or is it too much work for one PP?

Reply:
I can add a TODO here and do it in a follow-up to make it easier to review

Reply:
I added the error formatting to OM in my latest commit.